File: internal/astgrep/testdata/fixtures/csharp/suppressed.cs
// C# suppressed fixture — correctly paired ast-grep-ignore + @MX:REASON.
// checkSuppressionPairing should return 0 violations for this file.

public class Example
{
    public void Code()
    {
        // ast-grep-ignore
        // @MX:REASON test fixture for suppression policy; null assignment intentional for reset
        string x = null;
        _ = x;
    }
}

File: internal/astgrep/testdata/fixtures/elixir/suppressed.ex
# Elixir suppressed fixture — correctly paired ast-grep-ignore + @MX:REASON.
# checkSuppressionPairing should return 0 violations for this file.

defmodule Example do
  def code do
    # ast-grep-ignore
    # @MX:REASON test fixture for suppression policy; _unused assignment intentional for testing
    _unused = :test_value
    :ok
  end
end
